.github/workflows/publish.yml

```diff
@@ -35,10 +35,6 @@ jobs:
           python-version: "3.9"
           pandoc: "False"
 
-      - name: "Install gettext for translations 🌐"
-        run: |
-          sudo apt-get install gettext
-
```

Comment:
After looking through the entire diff, I still don't understand why this was removed

Reply:
We do not use gettext directly but instead we use pybabel to handle the translation-related tasks.
So we really do not need a separate install, this seems to have been left over from previous iterations of the translation workflows.